src/makeAuth/index.test.tsx

```diff
@@ -92,4 +94,19 @@ describe('makeAuthenticator', () => {
     fireEvent.click(button)
     waitForElement(() => expect(queryByText('Logout')).toBeNull())
   })
+  
+  it('has props passed through', async () => {
+    const userManagerConfig = {
+      getUserFunction: failedGetUser
```

Comment:
Let's change this to the `successfulGetUser`.

```suggestion
      getUserFunction: successfulGetUser
```

Currently your test asserts that `'bar'` is **not** present, which is correct if `getUserFunction` fails, but it is also correct if your implementation of passing props is broken, so this test does not strictly check that the passing of props works.

After this, you will need to update line 110. I've made a suggestion there.